backend/src/v5/utils/quota.js

```diff
@@ -59,7 +59,7 @@ Quota.getQuotaInfo = async (teamspace) => {
 };
 
 Quota.getSpaceUsed = async (teamspace) => {
-	const colsToCount = ['.history.ref', '.issues.ref', '.risks.ref', '.resources.ref'];
+	const colsToCount = ['.history.ref', '.issues.ref', '.risks.ref', '.resources.ref', 'tickets.resources.ref'];
```

Comment:
judging by this logic, it was already using taking into account tickets usage? Didn't seem like the change was necessary afterall

Reply:
ah true